src/Controller/ComptableController.php

```diff
@@ -18,7 +19,7 @@ class ComptableController extends AbstractController
 
     #[Route('/', name: 'app_comptable_index', methods: ['GET', 'POST'])]
```

Comment:
Why do we accept POST method in this route

Reply:
No route found for "POST http://127.0.0.1:8000/comptable/": Method Not Allowed (Allow: GET)

Parceque j'ai cette erreur quand je fais un call au serveur avec une date